trace/LogCorrelation.md

```diff
@@ -0,0 +1,70 @@
+# Log Correlation
+
+Log correlation is a feature that inserts information about the current span into log entries
+created by existing logging frameworks.  The feature can be used to add more context to log entries,
+filter log entries by trace ID, or show log entries as annotations on a trace.
+
+The design of a log correlation implementation depends heavily on the details of the particular
+logging framework that it supports.  Therefore, this document only covers the aspects of log
+correlation that could be shared across log correlation implementations for multiple languages and
+logging frameworks.  It doesn't cover how to hook into the logging framework.
+
+## Identifying the span to associate with a log entry
+
+A log correlation implementation should look up tracing data from the span that is current at the
+point of the log statement.  See
+[Span.md#how-span-interacts-with-context](Span.md#how-span-interacts-with-context) for the
+definition of the current span.
+
+## Tracing data to include in log entries
+
+A log correlation implementation should insert the following pieces of tracing data from the current
+span context into each log entry:
```

Comment:
make available in the logging entry. eg in some systems it might not literally be a copy rather sharing in context.